app/src/main/res/layout/activity_show_ticket.xml

```diff
@@ -35,40 +35,33 @@
                     style="@style/SmallListSeparatorTextViewStyle"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:text="@string/moviesdetail_dots"
-                    android:textSize="16sp"/>
+                    android:text="@string/eventdetail_dots"
+                    android:textSize="16sp" />
 
                 <TextView
-                    android:id="@+id/moviedetail"
+                    android:id="@+id/eventdetail"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_margin="16dp"
                     android:text=""
-                    android:textSize="@dimen/large_text_size"/>
+                    android:textSize="@dimen/large_text_size" />
 
                 <TextView
                     style="@style/SmallListSeparatorTextViewStyle"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:text="@string/ticketnumber_dots"
-                    android:textSize="16sp"/>
+                    android:text="@string/qrcode_dots"
+                    android:textSize="16sp" />
 
-                <TextView
-                    android:id="@+id/ticketnumber"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_margin="16dp"
-                    android:text=""
-                    android:textSize="@dimen/large_text_size"/>
                 <ImageView
                     android:id="@+id/ticket_qrcode"
                     android:layout_width="match_parent"
                     android:layout_height="match_parent"
```

Comment:
When you rotate the device, the QR gets too small to be used, because the layout also in landscape mode tries to fit everything height-wise into the screen.
It would be nice, if the QR code was also usable in landscape mode. Maybe you therefore need to put a changed version of the layout in the "layout-land" directory.